.github/workflows/sync-kuma.yml

```diff
@@ -0,0 +1,35 @@
+name: "Sync KUMA"
+
+on:
+  push:
+    branches:
+      - "main"
+  schedule:
+    - cron: "0 2 * * *"
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Login to Docker Hub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_PASSWORD }}
+    - name: Login to SIGHUP new Registry
+      uses: docker/login-action@v1
+      with:
+        registry: registry.sighup.io
+        username: ${{ secrets.SIGHUP_REGISTRY_USERNAME }}
+        password: ${{ secrets.SIGHUP_REGISTRY_PASSWORD }}
+    - name: Install yq
+      run: |
+        sudo apt-get update && sudo apt-get install -yqq wget
+        sudo wget -q https://github.com/mikefarah/yq/releases/download/v4.19.1/yq_linux_amd64 -O /usr/bin/yq
+        sudo chmod +x /usr/bin/yq
+    - name: Iterate
+      run: |
+        yq --version
+        docker run --rm quay.io/skopeo/stable:v1.13 --version
+        ./single_sync.sh modules/kuma/images.yml false
```

Comment:
Can you change this to use the sync v2?

Check monitoring workflow https://github.com/sighupio/fury-distribution-container-image-sync/blob/main/.github/workflows/sync-monitoring.yml